tools/osquery/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '2'
 
 services:
   ubuntu14-osquery:
-    image: "kolide/osquery:${KOLIDE_OSQUERY_VERSION}"
+    image: "dactiv/osquery:4.3.0-ubuntu14.04"
```

Comment:
Given that this is in the `kolide` org, it may make more sense to fix `kolide/osquery`? Thoughts

Reply:
If you would like to maintain `kolide/osquery`, that's fine with me. The important bit IMO is that we are developing with new versions of osquery.

Reply:
@directionless do you think Kolide can take on maintaining `kolide/osquery`?

Reply:
Maybe. Want to help? I can give you permission on both the git side, and the docker side...